### ISSUE TITLE
mmaction2_tutorial.ipynb error

### ISSUE THREAD

**Issue by Ishihara-Masabumi** [open-mmlab/mmaction2#2371]
### The doc issue

At Train a new recognizer in mmaction2_tutorial.ipynb, the following error occurred.

```
import os.path as osp
import mmengine
from mmengine.runner import Runner

# Create work_dir
mmengine.mkdir_or_exist(osp.abspath(cfg.work_dir))

# build the runner from config
runner = Runner.from_cfg(cfg)

# start training
runner.train()
.
.
.
after_run:
(BELOW_NORMAL) LoggerHook                         
 -------------------- 
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
[/usr/local/lib/python3.9/dist-packages/mmengine/registry/build_functions.py](https://localhost:8080/#) in build_from_cfg(cfg, registry, default_args)
    120             else:
--> 121                 obj = obj_cls(**args)  # type: ignore
    122 

9 frames
ValueError: batch_size should be a positive integer value, but got batch_size=0

During handling of the above exception, another exception occurred:

ValueError                                Traceback (most recent call last)
[/usr/local/lib/python3.9/dist-packages/mmengine/registry/build_functions.py](https://localhost:8080/#) in build_from_cfg(cfg, registry, default_args)
    133             cls_location = '/'.join(
    134                 obj_cls.__module__.split('.'))  # type: ignore
--> 135             raise type(e)(
    136                 f'class `{obj_cls.__name__}` in '  # type: ignore
    137                 f'{cls_location}.py: {e}')

ValueError: class `EpochBasedTrainLoop` in mmengine/runner/loops.py: batch_size should be a positive integer value, but got batch_size=0
```

Please let me know how to fix it.

**Comment by cir7** [comment]
@Ishihara-Masabumi The issue is due to an unexpected modification on config file. Adjust the batchsize as the error raised will resolve the issue. We will fix it asap.

**Comment by cir7** [comment]
This problem has been fixed in #2375 and #2384 . We will close the issue, if you have any further questions, feel free to reopen it.

**Comment by Ishihara-Masabumi** [comment]
OK, thanks.
